src/main/webapp/app/course/manage/detail/course-detail-line-chart.component.ts

```diff
@@ -186,4 +192,82 @@ export class CourseDetailLineChartComponent extends ActiveStudentsChart implemen
         this.showAverage = !this.showAverage;
     }
+
+    /**
+     * Creates the chart for the default scope (<= 17 weeks)
+     */
+    displayDefaultChartScope(): void {
+        this.showLifetimeOverview = false;
+        this.loading = true;
+        // Only use the pre-loaded stats once
+        if (!this.initialStats) {
+            return;
+        }
+        this.createLabels();
+        this.processDataAndCreateChart(this.initialStats);
+        this.data = this.dataCopy;
+    }
+
+    /**
+     * Creates the chart for the lifetime overview
+     */
+    displayLifetimeOverview(): void {
+        // if the course does not have a start date, we can't create an meaningful lifetime overview
+        if (!this.course.startDate) {
+            return;
+        }
+        this.showLifetimeOverview = true;
+        this.loading = true;
+        this.currentPeriod = 0;
+        this.startDateDisplayed = true;
+        this.showsCurrentWeek = true;
+        this.createLabels();
+        // we cache the overview stats to prevent many server requests
+        if (!this.overviewStats) {
+            this.fetchLifetimeOverviewData();
+        } else {
+            this.processDataAndCreateChart(this.overviewStats);
+            this.data = [...this.dataCopy];
+        }
+    }
+
+    /**
+     * Auxiliary method reducing the complexity of {@link CourseDetailLineChartComponent#createLabels}
+     * Assigns the correct calendar week numbers to the corresponding data objects as string
+     * Note: the conversion to strings is important as ngx-charts increases the tick steps of on the x axis otherwise
+     * @private
+     */
+    private assignLabelsToDataObjects(): void {
+        let currentWeek;
+        for (let i = 0; i < this.currentSpanSize; i++) {
+            currentWeek = dayjs()
+                .subtract(this.currentOffsetToEndDate + this.currentSpanSize - 1 - this.displayedNumberOfWeeks * this.currentPeriod - i, 'weeks')
+                .isoWeekday(1)
+                .isoWeek();
+            this.dataCopy[0].series[i] = {};
+            this.dataCopy[0].series[i]['name'] = currentWeek.toString();
+            this.absoluteSeries[i] = {};
+            this.absoluteSeries[i]['name'] = currentWeek.toString();
+        }
+    }
+
+    /**
+     * Fetches and caches the data for the lifetime overview from the server and creates the chart
+     * @private
+     */
+    private fetchLifetimeOverviewData(): void {
+        this.service.getStatisticsForLifetimeOverview(this.course.id!).subscribe((res: number[]) => {
+            this.overviewStats = res;
+            this.processDataAndCreateChart(this.overviewStats);
+            this.data = [...this.dataCopy];
+        });
+    }
+
+    /**
+     * Auxiliary method handles the translation sensitivity of the x axis label
+     * @private
+     */
+    private updateYAxisLabel() {
+        this.xAxisLabel = this.translateService.instant('artemisApp.courseStatistics.calendarWeek');
```

Comment:
The method is called `updateYAxisLabel`, but updates the `X` axis label.

Reply:
Good catch, might be a bit confused there. Changed in 7459fd4.